### DISCUSSION TITLE
changing home WiFi settings at runtime

### DISCUSSION THREAD

**Discussion by janjos0** [easytarget/esp32-cam-webserver#301] [Ideas]
Hi,

First, thank you Owen Carter for this very usable firmware.

I have been using this firmware for a week now together with ContaCam, https://www.contaware.com/contacam.html. 
Up to now it is not the best combination as I get many NO FRAMES message in the stream video window of ContaCam.
And I only use 640 x 480 image size and 5 FPS.
I am almost sure the problem is ContaCam and not the esp32-cam webserver, because in a browser window the stream is fluent and does not seem to have any frames dropped.

Now to my question, that maybe it is already implemented but I did not find how, yet,
is it possible to change during runtime the home WiFi settings (ssid and password) the ESP32-CAM is connected to ?
The server would disconnect from the WiFi signal is connected to and forget about it and when found sign in to the new ssid.
Why, because at home I have 2 WiFi signals that overlap a bit at the periphery of their range and would like to switch from one to the other without having to rewrite the firmware with the settings of one or the other.
I know I can input multiple ssid's in myconfig.h but I only program the one I want to connect to depending where the board will be located in the house.

**Comment by easytarget** [comment]
Not really possible with the code as-is, it will select the best signal from the list of matching SSID's.

I am not doing any further development on this project. it is old code and somewhat obsolete. I am currently working on retiring it; I will try to link to a more actively updated fork when I do.
